locator/ec2_snitch.hh

```diff
@@ -13,6 +13,7 @@
 namespace locator {
 class ec2_snitch : public production_snitch_base {
 public:
+    static constexpr const char* TOKEN_REQ_ENDPOINT = "/latest/api/token";
```

Comment:
So, AWS wants you to fetch the token from AWS, and then send the token back to AWS?

Reply:
As weird as it sounds, yes. And if you don't do so, and one configures the instance to require a HTTP token, a direct access without specifying a token (to avoid SSRF) you'll get a HTTP 401 (Unauthorized) response

The reasoning is to avoid this "threat": https://www.uptycs.com/blog/you-should-be-using-aws-imdsv2-heres-why-and-how-to-do-it

Reply:
I see, so instead of forging one URL, the attacker has to forge two URLs.

Reply:
Makes (some) sense.